app/Example.java

```diff
@@ -0,0 +1,10 @@
+import io.ably.lib.realtime.AblyRealtime;
+
+public class Example {
+
+	private static final String API_KEY = "INSERT-YOUR-API-KEY-HERE";
```

Comment:
Same as other PR, raise an exception if this is not defined as the user is then running this mistakenly